charts/admission-controller/templates/webhook/admissionregistration.yaml

```diff
@@ -1,3 +1,19 @@
+{{- if .Values.webhook.ssl.certManager.enabled }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "admissionController.webhook.fullname" . }}
+    helm.sh/hook: "post-install, post-upgrade"
+    meta.helm.sh/release-name: {{ .Release.Name }}
+    meta.helm.sh/release-namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: {{ include "admissionController.webhook.fullname" . }}
+  namespace: {{ include "admissionController.namespace" . }}
+{{- include "admissionController.webhookTemplate" . }}
```

Comment:
Note that I don't inject any `Cert` value here so the resulting `ValidatingWebhookConfiguration` manifest won't contain any `caBundle` as per the `_helper.tpl` changes above.